packages/react-notion-x/src/styles.css

```diff
@@ -1600,6 +1600,323 @@ svg.notion-page-icon {
+  border-right: 1px solid rgb(233, 233, 231);
+  border-bottom: 1px solid rgb(233, 233, 231);
+  cursor: default;
+}
+
+.notion-calendar-body-inner-week-dif {
+  background: rgb(251, 251, 250);
+}
+
+.notion-calendar-body-inner-day {
+  position: absolute;
+  font-size: 14px;
+  top: 4px;
+  right: 10px;
+  height: 24px;
+  line-height: 24px;
+}
+
+.notion-calendar-body-inner-day-today {
+  width: 24px;
+  border-radius: 100%;
+  text-align: center;
+  color: white;
+  background: rgb(235, 87, 87);
+}
+
+.notion-calendar-body-inner-day-this-month {
+  color: black;
+  text-align: right;
+  transition: color 100ms ease-out 0s;
+}
+
+.notion-calendar-body-inner-day-other-month {
+  color: rgba(55, 53, 47, 0.5);
+  text-align: right;
+  transition: color 100ms ease-out 0s;
+}
+
+.notion-calendar-body-inner-card {
+  width: calc(14.2857%);
+  position: absolute;
+  padding: 3px 6px;
+  top: 30px;
+}
+
+.notion-calendar-body-inner-card-inner {
+  display: block;
+  color: inherit;
+  text-decoration: none;
+  height: 100%;
+  background: white;
+  border-radius: 3px;
+  box-shadow: rgba(15, 15, 15, 0.1) 0px 0px 0px 1px,
+    rgba(15, 15, 15, 0.1) 0px 2px 4px;
+}
+
+.notion-calendar-body-inner-card-inner-box {
+  user-select: none;
+  transition: background 20ms ease-in 0s;
+  cursor: pointer;
+  width: 100%;
+  display: flex;
+  position: relative;
+  padding-top: 2px;
+  padding-bottom: 2px;
+  height: 100%;
+  align-items: flex-start;
+  flex-direction: column;
+}
+
+.notion-calendar-body-inner-card-inner-box:hover {
+  background: rgba(55, 53, 47, 0.08);
+}
+
+.notion-calendar-body-inner-card-inner-box-title {
+  padding-left: 6px;
+  padding-right: 6px;
+  overflow: hidden;
+  width: 100%;
+  font-size: 12px;
+}
+
+.notion-calendar-body-inner-card-inner-box-title-inner {
+  display: flex;
+  align-items: center;
+  height: 20px;
+}
+
+.notion-calendar-body-inner-card-inner-box-title-inner-icon {
+  user-select: none;
+  transition: background 20ms ease-in 0s;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 12px;
+  width: 12px;
+  border-radius: 0.25em;
+  flex-shrink: 0;
+  margin-right: 4px;
+  margin-top: 2px;
+}
+
+.notion-calendar-body-inner-card-inner-box-title-inner-icon-svg {
+  width: 10.8px;
+  height: 10.8px;
+  display: block;
+  fill: rgba(55, 53, 47, 0.85);
+  flex-shrink: 0;
+  backface-visibility: hidden;
+}
+
+.notion-calendar-body-inner-card-inner-box-title-inner-text {
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  flex-grow: 1;
+  font-size: 12px;
+  font-weight: 600;
+}
+
+.notion-calendar-body-inner-card-inner-box-properties {
+  padding-left: 6px;
+  padding-right: 6px;
+  overflow: hidden;
+  width: 100%;
+}
+
+.notion-calendar-body-inner-card-inner-box-properties-property {
+  display: flex;
+  align-items: center;
+  font-size: 12px;
+  height: 20px;
+  white-space: nowrap;
+}
+
+.dark-mode .notion-calendar-header {
+  background: rgb(25, 25, 25);
```

Comment:
Ideally, these dark mode colors would be css variables defined up top, so it's easy to change the theme via only css vars. Not a big deal; just a note.